configuration/analyzer_config.json

```diff
@@ -772,6 +772,20 @@
       }
     }
   },
+  "Filescan": {
```

Comment:
Rename to `FileScan_Upload_File`.

Reply:
Also you've to append this analyzer name [here](htatps://github.com/intelowlproject/IntelOwl/blob/3d860e5279e49aee583cd9b4c48e28f4cc894fef/tests/analyzers_manager/test_file_scripts.py#L56) so this analyzer is run in the CI tests.